src/Tribe/Rewrite.php

```diff
@@ -255,11 +255,12 @@ public function get_bases( $method = 'regex' ) {
 			'tag' => array( 'tag', $tec->tag_slug ),
 			'tax' => array( 'category', $tec->category_slug ),
 			'page'     => array( 'page', esc_html_x( 'page', 'The "/page/" URL string component.', 'the-events-calendar' ) ),
-			'single' => array( 'event', $tec->rewriteSlugSingular ),
-			'archive' => array( 'events', $tec->rewriteSlug ),
+			'single' => array( Tribe__Settings_Manager::get_option( 'singleEventSlug', 'event' ), $tec->rewriteSlugSingular ),
```

Comment:
This improves things but as I think about it, shouldn't we do something like:

```php
array_filter( array(
	'event',
	Tribe__Settings_Manager::get_option( 'singleEventSlug', 'event' ),
	$tec->rewriteSlugSingular
) )
```

Looking at the comment above this block, we always want to support the original/English slug - but right now we potentially will miss that element.

Reply:
Uhnn I was thinking maybe the intention was to allow the users to completely remove `events/event` using a different option on the DB.

Reply:
This one should be only for the single view? as if we do it for `archive`  we are still going to have the same issue if we leave `events` @barryhughes

Reply:
Hmm, don't think it is but if I'm wrong let's also update the comment above this section.

Seems though that we'd be creating a headache in scenarios where URLs using the original slug are promulgated via sharing links etc.

Reply:
> This one should be only for the single view?

Reply:
if we leave `events` hardcoded is going to be present anyways  so we are going to have the `events` url taken by the plugin so I think we should update the documentation around this, plus is the default value if the option value is not changed if will be different only if the user change is intentionality. 

cc @bordoni @barryhughes

Reply:
I think the sane default is for us to support:

Reply:
... filtering out any duplicates

Reply:
Actually, I see what you're saying. Let's keep this as is.